app/AccountancyModule/PaymentModule/components/templates/SmtpAddForm.latte

```diff
@@ -0,0 +1,41 @@
+<div class="col-lg-5">
```

Comment:
Ta definice sloupce dává větší smysl přímo v `Email/default.latte`.

Reply:
to určitě

Reply:
@sinacek Ještě tohle přesunout do té hlavní šablony akce

Reply:
měl jsem to hotove a zapomněl to commitnout